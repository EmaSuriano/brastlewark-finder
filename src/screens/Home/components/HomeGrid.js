import styled from 'styled-components';

const HomeGrid = styled.div`
  display: grid;
  justify-items: center;
  grid: auto auto 1fr/ 1fr;
  justify-content: center;
  max-width: 1500px;
  align-items: center;
  margin: auto;
  padding: 20px;
`;

export default HomeGrid;
