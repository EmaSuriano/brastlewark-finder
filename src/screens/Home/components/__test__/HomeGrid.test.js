import React from 'react';
import { shallow } from 'enzyme';
import HomeGrid from '../HomeGrid';

describe('<HomeGrid />', () => {
  it('render', () => {
    expect(shallow(<HomeGrid />)).toMatchSnapshot();
  });
});
