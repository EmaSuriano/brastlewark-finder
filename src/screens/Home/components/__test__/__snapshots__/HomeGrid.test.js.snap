// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeGrid /> render 1`] = `
.c0 {
  display: grid;
  justify-items: center;
  grid: auto auto 1fr/ 1fr;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  max-width: 1500px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: auto;
  padding: 20px;
}

<div
  className="c0"
/>
`;
